frontend/src/pages/modelRegistry/screens/ModelVersionDetails/const.ts

```diff
@@ -7,3 +7,9 @@ export enum ModelVersionDetailsTabTitle {
   DETAILS = 'Details',
   DEPLOYMENTS = 'Deployments',
 }
+
+export const pipelineRunSpecificKeys: string[] = [
+  '_registeredFromPipelineProject',
+  '_registeredFromPipelineRunId',
+  '_registeredFromPipelineRunName',
+];
```

Comment:
If we're simply filtering out by `_registeredFrom*` and we're writing a `getRegisteredFromPipelineProperties` util, maybe we can just reference these property keys directly inside that util and not expose them anywhere else (we could remove this export). Then we can drive everything from the return values of that util and it'll probably be easier to refactor later to use different properties.